src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkHeader.tsx

```diff
@@ -17,28 +17,24 @@ export const MyCollectionArtworkHeader: React.FC<
     useMyCollectionTracking()
 
   const artwork = useFragment(FRAGMENT, props.artwork)
-  const showEditCTA = !artwork.consignmentSubmission?.internalID
```

Comment:
Ahh – it's interesting that this was more restrictive than the condition over in Eigen which referred to [Metaphysics's isEditable field](https://github.com/artsy/metaphysics/blob/ad0a5503044af163d8702afe4742da25d0450b3a/src/schema/v2/artwork/artworkConsignmentSubmissionType.ts#L40-L47) – based on the submission status.